.github/workflows/test.yml

```diff
@@ -8,7 +8,7 @@ on:
     - cron: '0 */12 * * *'
 
 env:
-  GO_VERSION: 1.15.5
+  GO_VERSION: 1.16.2
```

Comment:
we have to remove the GOPATH sooner or later, so let's try to do it now, to see if is the problem
```diff
diff --git a/.github/workflows/test.yml b/.github/workflows/test.yml
index b67a4ea0..15a3f2e8 100644
--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -56,22 +56,17 @@ jobs:
       run: |
         set -x
         go get golang.org/x/tools/cmd/cover
+        go get github.com/modocache/gover
+        go get github.com/jandelgado/gcov2lcov
         pushd go-controller
            make
            make windows
            COVERALLS=1 make check
+           # Combine separate code coverage profiles into one
+           gover go-controller/ gover.coverprofile
+           # Convert coverage profile to LCOV format for coveralls github action
+           gcov2lcov -infile gover.coverprofile -outfile ../coverage.lcov
         popd
-        # Combine separate code coverage profiles into one
-        go get github.com/modocache/gover
-        gover go-controller/ gover.coverprofile
-
-        # Convert coverage profile to LCOV format for coveralls github action
-        go get github.com/jandelgado/gcov2lcov
-        mkdir -p src/github.com/ovn-org
-        ln -sf $(pwd) src/github.com/ovn-org/ovn-kubernetes
-        GOPATH=$(pwd) gcov2lcov -infile gover.coverprofile -outfile coverage.lcov
-
     - name: Build docker image
       run: |
         pushd dist/images
```